python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/router_bgp.schema.yml

```diff
@@ -2769,36 +2631,17 @@ keys:
                             - "permit"
                     additional_paths:
                       type: dict
+                      $ref: "eos_cli_config_gen#/$defs/additional_paths"
                       keys:
                         install:
                           type: bool
+                          description: Install BGP backup path.
                         install_ecmp_primary:
                           type: bool
-                        receive:
-                          type: bool
-                        send:
-                          type: dict
-                          keys:
-                            any:
-                              type: bool
-                            backup:
-                              type: bool
-                            ecmp:
-                              type: bool
-                            ecmp_limit:
-                              type: int
-                              description: Amount of ECMP paths to send.
-                              convert_types:
-                                - str
-                              min: 2
-                              max: 64
-                            limit:
-                              type: int
-                              description: Amount of paths to send.
-                              convert_types:
-                                - str
-                              min: 2
-                              max: 64
+                          description: Allow additional path with ECMP primary path.
+                        prefix_list:
```

Comment:
Please not that this address-family is under vrf and it does not support prefix list

Reply:
s1-leaf1(config-router-bgp)#sh run | s vrf abc
router bgp 65000
   vrf abc
      address-family ipv4
         neighbor 10.10.10.10 additional-paths send any prefix-list abc
s1-leaf1(config-router-bgp)#

Reply:
Removed prefix-list from vrf.address_family_ipv4/v6.bgp. Added it under neighbors.